src/firebase.js

```diff
@@ -0,0 +1,15 @@
+import * as firebase from 'firebase';
+
+const config = {
+    apiKey: process.env.REACT_APP_API_KEY,
```

Comment:
You'll need to make sure that your `.env` file is in the root folder of the project, and that the variables names match up exactly, eg;
```
REACT_APP_API_KEY=abc123
```

I also found that when I was editing the `.env` file I had to stop and restart the dev server each time.

Reply:
Yes, I checked that every variable name matches up exactly.